aspnetcore/fundamentals/configuration/index/samples/8.x/ConfigSample/Options/NameTitleOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace ConfigSample.Options;
+
+
+public abstract class SomethingWithAName
+{
+    public abstract string? Name { get; set; }
+}
+
+public class NameTitleOptions(int age) : SomethingWithAName
+{
+    public const string NameTitle = "NameTitle";
+
+    public override string? Name { get; set; }
+    public string Title { get; set; } = string.Empty;
+
+    public int Age { get; set; } = age;
+}
```

Comment:
@SteveDunn your code From [this comment](https://github.com/dotnet/AspNetCore.Docs/pull/30842#pullrequestreview-1702713664)